Source/Task/WaitTimer_win32.cpp

```diff
@@ -94,6 +97,16 @@ HRESULT WaitTimer::Initialize(_In_opt_ void* context, _In_ WaitTimerCallback* ca
     return S_OK;
 }
 
+void WaitTimer::Terminate() noexcept
+{
+    WaitTimerImpl* impl = static_cast<WaitTimerImpl*>(InterlockedExchangePointer(reinterpret_cast<volatile PVOID*>(&m_impl), nullptr));
```

Comment:
should we use atomics for all accesses to m_impl? should we do the same for the stl timer?

(the fact we aren't using any "thread safe" access for m_impl right now worries me)

Reply:
We could but it's not required because these are only cleaned up in the port dtor.  Since that is already ref-counted it's guarded there.

Reply:
Ok, moved this to an atomic in case this timer is used outside the task queue code.